### PR TITLE
Create anthony.json

### PR DESCRIPTION
**Pull request by greatbyte**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] The JSON file follows the template at [https://github.com/jscraftcamp/website/blob/main/participants/\_template.json](https://jscraftcamp.org/register/) and contains the mandatory fields like `name`, `when`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

**Comment by meldron** [comment]
Hi @greatbyte, 

there are some problems with your PR. First you forgot to tick the check marks inside the PR Text. See the [Github Markdown Documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#task-lists) on how you can tick the check boxes.

Also your **twitter** handle cannot start with the `@` sign, just remove it.
Your **mastodon** handle must be a valid URL to your profile.

If you need any additional help let us know.

File: participants/anthony.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "saturday": false
+    },
+    // required
+    "iCanTakeNotesDuringSessions": false,
+    // required - at least one
+    "tags": ["ReScript"],
+    // optional
+    "vegan": true/false,
+    // optional
+    "vegetarian": true/false,
```
Comment (meldron) [inline participants/anthony.json:18]:
```suggestion
    // optional
    "vegan": false,
    // optional
    "vegetarian": true,
```

You have to select a boolean value here

**Comment by meldron** [comment]
Hi @greatbyte, please adjust your changes in this branch of your fork. I closed the other PR because it had way to many changes.

**Comment by greatbyte** [comment]
I have edited the boolean values and hope everything is correct.

**Comment by greatbyte** [comment]
I did the corrections and hope it is fine now. Thanks.

Anthony

File: participants/anthony.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    // required
+    "name": "Anthony Craig",
+    // optional
+    "company": "your company name",
```
Comment (meldron) [inline participants/anthony.json:5]:
```suggestion
    "company": "your company name",
```

Can you update your company name? If you don't want to mention one, feel free to just remove this property.

File: participants/anthony.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    // required
+    "iCanTakeNotesDuringSessions": false,
+    // required - at least one
+    "tags": ["ReScript"],
+    // optional
+    "vegan": false,
+    // optional
+    "vegetarian": true,
+    // optional
+    "allergies": ["Gluten", "Peanuts"],
```
Comment (meldron) [inline participants/anthony.json:20]: Are you really allergic to both our sample allergy options?

**Review by meldron — CHANGES_REQUESTED** [review]

File: participants/anthony.json (hunk 3 above)
Comment (greatbyte) [reply participants/anthony.json]: I am not allergic to both

File: participants/anthony.json (hunk 2 above)
Comment (greatbyte) [reply participants/anthony.json]: I do not want to mention one.

**Review by meldron — APPROVED** [review]

**Comment by greatbyte** [comment]
Hi,
If I am allowed for both days would be great.

Tech/tags:

CS50, React, vanilla javascript.

Thanks,

Anthony

File: participants/anthony.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+    // required
+    "name": "Anthony Craig",
+    
+    // required
+    "when": {
+        "friday": true,
+        "saturday": false
```
Comment (Narigo) [inline participants/anthony.json:8]:
```suggestion
        "saturday": true
```

File: participants/anthony.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    // required
+    "iCanTakeNotesDuringSessions": false,
+    // required - at least one
+    "tags": ["ReScript"],
```
Comment (Narigo) [inline participants/anthony.json:13]:
```suggestion
    "tags": ["CS50", "React", "Vanilla JS"],
```

Comment (Narigo) [inline participants/anthony.json:13]:
```suggestion
    "tags": ["CS50", "React", "Vanilla JS"],
```

**Comment by meldron** [comment]
> Hi, If I am allowed for both days would be great. Tech/tags: CS50, React, vanilla javascript. Thanks, Anthony

Yeah sure, just adjust your PR.

**Review by Narigo — COMMENTED** [review]
Hi @greatbyte !
Based on your last comment, I have added some suggestions. Please commit them by using the "Commit suggestion", if you want to change the preferences accordingly.

I'm not sure about the `vegan` / `vegetarian` preferences as you have used the suggestion by @meldron without commenting anything about it.

**Comment by greatbyte** [comment]
My preference is fluid. But being vegetarian is OK with me.

**Comment by meldron** [comment]
Hey @greatbyte, welcome to JSCC 23! Looking forward to meeting you there!

**Comment by greatbyte** [comment]
Thanks.

**Pull request merged**